### ISSUE TITLE
RFDETR medium ONNX export taking too long

### ISSUE THREAD

**Issue by aabbhishekksr** [roboflow/rf-detr#281]
### Search before asking

- [x] I have searched the RF-DETR issues and found no similar bug report.

### Bug

While I am following documentation for fine-tuning on a custom dataset,

I finetuned the medium variant and then , as per documentation trying to export to ONNX, but it is kind of taking too much of time .

what could be the issue here
I can see following output
**Exporting model to ONNX format
PyTorch inference output shapes - Boxes: torch.Size([1, 3900, 4]), Labels: torch.Size([1, 3900, 12])**

### Environment

RF-DETR 1.2.1
OS Ubuntu
Python 3.10

### Minimal Reproducible Example

from rfdetr import RFDETRMedium
model = RFDETRMedium(pretrain_weights='output/checkpoint_best_total.pth')
model.optimize_for_inference()
model.export()

### Are you willing to submit a PR?

- [ ] Yes, I'd like to help by submitting a PR!

**Comment by isaacrob-roboflow** [comment]
Can you provide your onnxruntime and PyTorch versions
